locales/es/translation.json

```diff
@@ -1,1327 +1,1328 @@
+    "Auto": "Auto",
+    "Automated process cost": "Coste del proceso automatizado",
+    "Automatically enable this rulebook activation to run.": "Habilitar automáticamente la activación de este reglamento para que se ejecute.",
+    "Automation": "Automatización",
+    "Automation Analytics is currently not enabled to send data.": "Actualmente, Automation Analytics no está habilitado para enviar datos.",
+    "Automation Content": "Contenido de automatización",
+    "Automation formula": "Fórmula de automatización",
+    "Automation savings": "Ahorros en automatización",
+    "Automation server": "Servidor de automatización",
+    "Automation servers": "Servidores de automatización",
+    "automation time = successful elapsed total / 3600 automation cost = automation time * configurable automated process cost": "tiempo de automatización = total transcurrido con éxito / 3600 coste de automatización = tiempo de automatización * coste del proceso automatizado configurable",
+    "Automation type": "Tipo de automatización",
+    "AWX": "AWX",
+    "AWX Ansible server": "Servidor AWX Ansible",
+    "Back to Credentials": "Volver a Credenciales",
+    "Back to Jobs": "Volver a Tareas",
+    "Back to Templates": "Volver a Plantillas",
+    "Back to Users": "Volver a Usuarios",
+    "basePathHelpBlock": "basePathHelpBlock",
+    "Being a team member grants the user all the permissions of the team.": "Ser miembro de un equipo otorga al usuario todos los permisos del equipo.",
+    "Branch to use in job run. Project default used if blank. Only allowed if project allow_override field is set to true.": "Rama para utilizar en la ejecución del trabajo. Se utiliza la configuración predeterminada del proyecto si está en blanco. Sólo se permite si el campo allow_override del proyecto está establecido en true.",
+    "brand logo": "logotipo de la marca",
+    "Brand Logo": "Logotipo de la marca",
+    "Browse all certified collections": "Ver todas las colecciones certificadas",
+    "Cache Timeout": "Tiempo de espera de la caché",
+    "Can approve or deny a workflow approval node": "Puede aprobar o denegar un nodo de aprobación de flujo de trabajo",
+    "can edit, change, and update any inventory or automation definition": "puede editar, cambiar y actualizar cualquier inventario o definición de automatización",
+    "Can manage all aspects of the credential": "Puede gestionar todos los aspectos de la credencial",
+    "Can manage all aspects of the inventory": "Puede gestionar todos los aspectos del inventario",
+    "Can manage all aspects of the job template": "Puede gestionar todos los aspectos de la plantilla",
+    "Can manage all aspects of the organization": "Puede gestionar todos los aspectos de la organización",
+    "Can manage all aspects of the project": "Puede gestionar todos los aspectos del proyecto",
+    "Can manage all aspects of the workflow job template": "Puede gestionar todos los aspectos del modelo de trabajo del flujo de trabajo",
```

Comment:
In the rest of places "workflow job template" is translated as "plantilla de trabajo del flujo de trabajo" so this should use it:
"Puede gestionar todos los aspectos de la plantilla de trabajo del flujo de trabajo"